_data/simple-icons.json

```
@@ -7551,6 +7551,11 @@
                 "type": "CC-BY-SA-4.0"
             }
         },
+        {
+            "title": "RxDb",
```

Though it doesn't really matter in the grand scheme of things, the brand is stylized as 'RxDB'.
```suggestion
            "title": "RxDB",
```